package.json

```diff
@@ -10,7 +10,9 @@
     "integration": "jest --config jest.integration.js --maxWorkers=2",
     "lint": "eslint . -c .eslintrc.yaml --no-eslintrc --ignore-path .gitignore --cache --cache-location ./node_modules/.cache/eslint",
     "pretest": "npm run lint",
-    "test": "jest"
+    "test": "jest",
+    "build": "npm run build:schema",
```

Comment:
When does the build need to run? `preversion`?

As it is now, this will never be run by any automated process, and humans (me) will inevitably forget to run it.

Reply:
Not sure where it should go exactly.  The schema should be bundled into the published tarball but it's not needed by the tests.  Could go in a `prepare` script.

Reply:
Seems like the schema should be located under the `core/lerna` directory, then, generated as a `prepublishOnly` script.